main.c

```diff
@@ -35,37 +31,26 @@ void selectionSort(int a[], int size)
 
 int main(int argc, char *argv[])
 {
-    int arr[argc-1];
```

Comment:
Very dangerous. This feature only exists in gcc. In general, array size cannot be set by a run-time known value. It must be a fixed constasnt like 10 or 2. If you want to create an array dynamically you have to call malloc.